ampd/ampd-release.md

```diff
@@ -11,19 +11,18 @@ This document outlines the step-by-step process for releasing a new version of A
 
 ## Release Process
 
-### 1. Update and Tag Release Version
+### 1. Get the release tag (Dry run)
 
 1. Navigate to the [Update and tag release version](https://github.com/axelarnetwork/axelar-amplifier/actions/workflows/release.yaml) GitHub Action.
 2. Run the workflow with the following:
    - Use workflow from: `main` branch
    - Enter `ampd` as the package input
-   - **IMPORTANT**: For the first run, enable "Dry run" to verify everything works correctly
+   - **IMPORTANT**: Enable "Dry run" to verify everything works correctly and make a note of the updated tag.
 
-3. After confirming the dry run completed successfully, run the workflow again with "Dry run" disabled to create the actual release tag.
 
-### 2. Update the Changelog
-
-1. Create a PR to update the root level `CHANGELOG.md` file based on the newly created tag from step 1.
+### 2. Ensure the Changelog is updated
+In general, Changelog should be kept up-to-date in every PR that makes changes to `ampd`.
+1. Create a PR to update the root level `CHANGELOG.md` file based on the new tag from step 1.
 2. Update the changelog header section with the new version information.
```

Comment:
add a step to create the release doc in the deployments repo if a non-trivial action needs to be taken by verifiers (i.e. it's not just restarting with the new binary). Chain integrations team should review and can communicate with verifiers